packages/docs/docs/getting-started/layouts.mdx

```diff
@@ -107,6 +107,65 @@ useful properties are listed below:
 <ApiSnippet url={'/api/2d/components/Layout#justifyContent'} />
 <hr />
 
+### Disabling `layout` on a child node
+
+Sometimes it can be useful to remove the node from the node tree. This way the
+node can be styled independently of its ancestor layout, as well as moved freely
```

Comment:
> node can be styled independently of its ancestor layout

A node can always be styled independently. The goal of disabling `layout` is to regain control over the node's position and size. Or, in other words, to exclude it from the parent's layout.